app/filtering.js

```diff
@@ -130,6 +130,14 @@ function registerForBeforeRequest (session, partition) {
         onBlockedInTor(details, muonCb)
         return
       }
+    } else {
+      const host = urlParse(details.url).host
```

Comment:
you probably want `hostname` not `host` since host may include a port number

Reply:
Ugh, thanks.  Where is this API documented?

Reply:
i find it easiest to just look at what it's mapped to in chromium's GURL in brave/common/extensions/url_bindings.cc

Reply:
but the API was intended to be consistent with Node's url.parse module